### DISCUSSION TITLE
[Weekly Curation Report]: Aug 5th - 11th

### DISCUSSION THREAD

**Discussion by fehrhart** [wikipathways/wikipathways-help#57] [curation-reports]
### Recent changes

Recent Changes: Some curation changes by established WP curators, some new pathways from my copy number variation project - thats it. 

### Under construction

Under Construction: The page in WP academy curation got apparently stuck... happened before but after reloading 3-4 times and still no move I gave up.

![image](https://github.com/wikipathways/wikipathways-help/assets/11389574/be63c102-78a2-4865-994a-f3904a9bba76)

### New pathways

New pathways: 
[Homo sapiens:PAFAH1B1 copy number variation](https://classic.wikipathways.org/index.php/Pathway:WP5409) ‎(9 August 2023 by [Fehrhart](https://classic.wikipathways.org/index.php/User:Fehrhart))
[Homo sapiens:15q25 copy number variation](https://classic.wikipathways.org/index.php/Pathway:WP5408) ‎(8 August 2023 by [Fehrhart](https://classic.wikipathways.org/index.php/User:Fehrhart))
[Homo sapiens:15q11q13 copy number variation](https://classic.wikipathways.org/index.php/Pathway:WP5407) ‎(8 August 2023 by [Fehrhart](https://classic.wikipathways.org/index.php/User:Fehrhart))
[Homo sapiens:13q12.12 copy number variation](https://classic.wikipathways.org/index.php/Pathway:WP5406) ‎(7 August 2023 by [Fehrhart](https://classic.wikipathways.org/index.php/User:Fehrhart))
[Homo sapiens:13q12 or CRYL1 copy number variation ](https://classic.wikipathways.org/index.php/Pathway:WP5405)‎(6 August 2023 by [Fehrhart](https://classic.wikipathways.org/index.php/User:Fehrhart))

### Deleted Pathways

Deleted pathways: I dont have deletions rights - tagged one created to check literature references. 

### Additional notes

Summer break - time for creation and curation :) 

I think it was mentioned in the slack - I swapped with Tooba for this week and she will take over mine end of August.

### Update Approved pathway tag

- [X] All Approved pathways tags were updated!

**Comment by khanspers** [comment]
Thanks @fehrhart! Great new pathways. I will delete the test pathway about literature references and attach the gpml directly to the GitHub issue for which it's relevant.

The broken **Under Construction** feed seems to be due to a problem with web services fetching the title of one of the pathways. I am investigating and will update here with any potential solution.

**Comment by khanspers** [comment]
Added a GitHub issue here: https://github.com/wikipathways/academy/issues/38
